maker-frontend/src/MakerApp.tsx

```diff
@@ -41,6 +42,8 @@ export default function App() {
     const order = useLatestEvent<Order>(source, "order", intoOrder);
     const walletInfo = useLatestEvent<WalletInfo>(source, "wallet");
     const priceInfo = useLatestEvent<PriceInfo>(source, "quote");
+    const takersOrUndefined = useLatestEvent<TakerId[]>(source, "takers");
+    let takers = takersOrUndefined ? takersOrUndefined! : [];
```

Comment:
There is a shortcut for this:

```suggestion
    let takers = takersOrUndefined || [];
```